waywiser_hwbringup/launch/rover.launch.py

```diff
@@ -137,62 +128,23 @@ def camera_conditional_launch(context):
         if 'enable_camera' in waywise_rover_node_params_dict:
             enable_camera = waywise_rover_node_params_dict['enable_camera']
 
-    camera_params_dict = {}
-    # Fix frame_id for depth camera point cloud: https://github.com/IntelRealSense/realsense-ros/tree/ros2-development?tab=readme-ov-file#ros2robot-vs-opticalcamera-coordination-systems # noqa
-    enable_pointcloud_tranformation = False
-    pointcloud_tranformation_params_dict = {}
+    camera_launch_acton = []
     if enable_camera:
-        with open(LaunchConfiguration('camera_config').perform(context)) as f:
-            camera_params_dict = yaml.safe_load(f)
-            if 'pointcloud_tranformation' in camera_params_dict:
-                pointcloud_tranformation_params_dict = camera_params_dict[
-                    'pointcloud_tranformation'
-                ]
-                if 'enable' in pointcloud_tranformation_params_dict:
-                    enable_pointcloud_tranformation = pointcloud_tranformation_params_dict[
-                        'enable'
+        camera_launch_acton = [
```

Comment:
I am not sure I agree that realsense_d435i.launch.py should be in waywiser_perception. Other than the realsense-specific stuff, waywiser_perception is hardware agnostic, isn't it? I would argue that we want all the hardware-specific stuff in waywiser_hwbringup. If hwbringup gets too big and messy, it could be divided into separate hw-specific packages.

Reply:
I understand that realsense_d435i.launch.py depends on waywiser_perception but it might still be nicer for waywiser_hwbringup to depend on waywiser_perception than waywiser_perception containing hw specifics?

Reply:
I agree. It makes sense to have the realsense_d435i.launch.py in waywiser_hwbringup, but I think the point_cloud_transformer would be appropriate to be within waywiser_perception as it is hw agnostic. I moved the files accordingly and tested them.